plugins/sudo/sudo.plugin.sh

```diff
@@ -0,0 +1,14 @@
+#! bash oh-my-bash.module
+
+add_sudo() {
+  local prefix suffix
+  prefix="${READLINE_LINE:0:$READLINE_POINT}"
+  suffix="${READLINE_LINE:$READLINE_POINT}"
+  if [[ -z $prefix ]] && [[ -z $sufix ]]; then
```

Comment:
```suggestion
  if [[ -z $prefix && -z $suffix ]]; then
```
- If we keep `prefix` and `suffix`, `sufix` is a typo of `suffix`.
- `&&` can be used in the conditional command.

Reply:
fixed in last commit